quipucords/tests/scanner/network/test_network_inspect.py

```diff
@@ -91,6 +92,22 @@ def setUp(self):
         self.stop_states = [ScanJob.JOB_TERMINATE_CANCEL, ScanJob.JOB_TERMINATE_PAUSE]
         self.interrupt = Mock(value=ScanJob.JOB_RUN)
 
+        self.cred_ssh_keyvalue = ("ACF9" * 12 + "\n") * 20
+        self.cred_ssh = Credential(
+            name="cred_ssh",
+            username="username2",
+            ssh_keyvalue=self.cred_ssh_keyvalue,
+        )
+        self.cred_ssh.save()
+        self.cred_ssh_data = model_to_dict(self.cred_ssh)
+
+        # setup source_ssh for scan
+        self.source_ssh = Source(name="source_ssh", port=22, hosts=["1.2.3.5"])
+        self.source_ssh.save()
+        self.source_ssh.credentials.add(self.cred_ssh)
+
+        self.host_ssh_list = [("1.2.3.5", self.cred_ssh_data)]
+
```

Comment:
while it's OK to have this here...
1. only the new tests depend on this setup: the "pytest way" to deal with it is creating a local fixture and have those tests consume it
2. both Credential and Source models have an equivalent factory boy... well, factory.

Take these as optional comments - especially 2.

Reply:
👍  @pytest.fixture ftw,  nice!!